src/server_manager/electron_app/package.action.sh

```diff
@@ -91,10 +92,11 @@ function main() {
       echo "Unknown option: ${i}"
       exit 1
       ;;
-    *) ;;
+    *) # only supports having one platform
+      PLATFORM="${i?Platform missing}"
```

Comment:
The error won't trigger, so you can remove it
```suggestion
      PLATFORM="${i}"
```